docs/index.md

```diff
@@ -1099,7 +1152,10 @@ combination method (only useful in `Decision Forests`):
     predicted if and only if the number of predictions for that category is
     at least the threshold value.
     Otherwise, the prediction is plurality for the rest of predicted
-    values.
+    values. This combination method will be deprecated in favour of the `votes`
+    kind of operating point. Please check the
+    [Operating point's predictions](operating-point's-predictions) section
```

Comment:
this link is wrong, no? it points to the section of op for models?

Reply:
no, it's not wrong. It refers to the models example and also mentions the end of the section after that.